docs/usage/getting_started.rst

```diff
@@ -82,23 +82,23 @@ biosamples config is a tsv separated file with the following columns
 #. Biosample 
 	- Name to associate with your sample. e.g K562
 #. DHS
-	- DNAse-seq BAM file (sorted w/ index file existence)
+	- DNAse-seq BAM file (sorted w/ .bai index file existence)
 #. ATAC
-	- ATAC-seq BAM/TagAlign file (sorted w/ index file existence)
+	- ATAC-seq BAM/TagAlign file (sorted w/ Tabix .tbi index file existence)
 #. H3K27ac
-	- H3K27ac ChIP seq BAM file
+	- H3K27ac ChIP seq BAM file (sorted w/ .bai index file existence)
 #. default_accessibility_feature
-	- Either DHS or ATAC
+	- Either DHS or ATAC (*explain what this means*)
 #. HiC_dir
-	- HiC directory for the biosample cell type. If not provided, uses avg hi-c
+	- HiC directory for the biosample cell type. If not provided and using hg38, uses a default cell-type average Hi-C file (*anthony check behavior here - since we have average hi-c only for hg38 and hg19, selecting a different genome build should result in not using any average hi-c data*)
```

Comment:
Actually, the default will be powerlaw if HiC isn't provided

"""
HiC directory for the biosample cell type. If not provided, uses powerlaw distance as a contact estimate
"""